src/vs/base/browser/ui/countBadge/countBadge.ts

```diff
@@ -47,7 +48,7 @@ export class CountBadge implements IThemable {
 		this.badgeBackground = this.options.badgeBackground;
 		this.badgeForeground = this.options.badgeForeground;
 		this.badgeBorder = this.options.badgeBorder;
-
+		this.container = container;
```

Comment:
The `CountBadge` does not own the `container`. It shouldn't do anything on it besides `appendChild`. We either set the `data-count` on `this.element` or we find a different way to fix this, if that doesn't work.

Reply:
@joaomoreno Hi, I have bind `count` to `this.element`, please review it.